src/LogikUI/Interop/Logic.cs

```diff
@@ -42,4 +64,9 @@ public struct CoolStruct
             public string ThisIsAnInterestingThing;
         }
     }
+
+    public struct Data
```

Comment:
This should either be changed to an opaque pointer or something similar so we don't need to use an unsafe context everywhere we want to use it.
I can also do this in a followup pr.